manager_f1tenth.py

```diff
@@ -5,9 +5,12 @@
 import json
 import atexit
 import subprocess
+import os
```

Comment:
I would like to use subprocess instead of os. I have a function called process error that can be used for shell commands.

Reply:
We cannot show suprocess becuase it doesn't take inputs from the user and hence doesn't let us interact with the environment

Reply:
@TylerMorton Actually I prefer os.system much more because it atleast prints out the status of the commands that are being run in the back ground.

Reply:
I believe du process can still do print. However, for the size of the project and how early it is, let's just have both operational.